src/index.js

```diff
@@ -279,42 +270,45 @@ function walk(exports, includeDefault) {
 function getMain({ options, entry, format }) {
 	const { pkg } = options;
 	const pkgMain = options['pkg-main'];
+	const pkgTypeModule = pkg.type === 'module';
+	let multipleEntries = options.multipleEntries;
```

Comment:
`multipleEntries` needs to act as an escape hatch to our name resolution logic to ensure no collisions.

This can be seen with Microbundle itself; when we create a `cli.js` and `microbundle.js`, we need to ensure the assigned name of `cli` does not get replaced by the `"main"` field ("microbundle").